cmd/gendeepcopy/deep_copy.go

```diff
@@ -79,7 +82,11 @@ func main() {
 		glog.Fatalf("error writing package line: %v", err)
 	}
 
-	versionPath := path.Join(pkgBase, group, version)
+	versionPath := path.Join(pkgBase, *groupVersion)
+	if group == "experimental" {
+		//TODO: we should rename the direcotry /expapi to /experimental, so directory path match groupVerson
```

Comment:
No. It's in the TODO list (https://github.com/kubernetes/kubernetes/issues/12413#issuecomment-130188046).